README.md

```diff
@@ -34,6 +34,7 @@ The examples folder contains example solutions across a variety of Google Cloud
 * [IoT Nirvana](examples/iot-nirvana) - An end-to-end Internet of Things architecture running on Google Cloud Platform.
 * [Kubeflow Pipelines Sentiment Analysis](examples/kubeflow-pipelines-sentiment-analysis) - Create a Kubeflow Pipelines component and pipelines to analyze sentiment for New York Times front page headlines using Cloud Dataflow (Apache Beam Java) and Cloud Natural Language API.
 * [Kubeflow Fairing Example](examples/kubeflow-fairing-example) - Provided three notebooks to demonstrate the usage of Kubeflow Faring to train machine learning jobs (Scikit-Learn, XGBoost, Tensorflow) locally or in the Cloud (AI platform training or Kubeflow cluster).
+* [Python CI/CD with Cloud Builder and CSR](examples/python-cicd-with-cloudbuilder)
```

Comment:
It looks like every other link has a brief description, this should have one too.

Reply:
Makes sense.